README.md

```diff
@@ -1,5 +1,25 @@
-Android User Interface Samples Repository
-=========================================
+# Android user interface samples
 
-This repository contains a set of individual Android Studio projects to help you get
-started writing/understanding Android user interface features.
+A collection of Android Studio projects to help you start building Android user interface features.
+
+## Installation
+
+Clone the repository, then open the samples as projects in Android Studio.
+
+## Support
+
+Stack Overflow: http://stackoverflow.com/questions/tagged/android
+
+If you find an error in the samples, please file an issue at https://github.com/android/user-interface.
```

Comment:
Should we update the link here to be the current `https://github.com/android/user-interface-samples`? It looks this one works, but redirects.

Reply:
Revised.